src/interfaces/ISablierMerkleFactory.sol

```diff
@@ -75,6 +72,10 @@ interface ISablierMerkleFactory is IAdminable {
                                  CONSTANT FUNCTIONS
     //////////////////////////////////////////////////////////////////////////*/
 
+    /// @notice Retrieves the fee for the provided campaign creator, using the default fee if no custom fee is set.
+    /// @param campaignCreator The address of the user for whom the fee is being computed.
+    function computeFee(address campaignCreator) external view returns (uint256);
```

Comment:
I don't really like the name of this function.

Nothing is 'computed' here. It's just a variable that gets retrieved from storage depending upon a boolean flag.

Can we please rename it to `getFee`?

Reply:
Sounds good.